.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
is it worth scoping the [pull_request events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) more?  I usually stick to `opened`, `edited`, and `synchronize`

Reply:
I don't usually bother.